unpaper fails cross compile, missing xsltproc

### Steps To Reproduce
Steps to reproduce the behavior:
1. `nix build .#pkgsCross.aarch64-multiplatform.unpaper`

### Build log
```
       > checking minix/config.h presence... no
       > checking for minix/config.h... no
       > checking whether it is safe to define __EXTENSIONS__... yes
       > checking for stdbool.h that conforms to C99... yes
       > checking for _Bool... yes
       > checking for library containing sqrt... -lm
       > checking pkg-config is at least version 0.9.0... yes
       > checking for LIBAV... yes
       > checking for xsltproc... no
       > configure: error: unable to find xsltproc, you need this to build man pages
```

--- a/pkgs/tools/graphics/unpaper/default.nix
+++ b/pkgs/tools/graphics/unpaper/default.nix
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, pkg-config, ffmpeg_4, libxslt }:
+{ lib, stdenv, fetchurl, buildPackages, pkg-config, ffmpeg_4 }:
 
 stdenv.mkDerivation rec {
   pname = "unpaper";
@@ -9,8 +9,8 @@ stdenv.mkDerivation rec {
     sha256 = "0c5rbkxbmy9k8vxjh4cv0bgnqd3wqc99yzw215vkyjslvbsq8z13";
   };
 
-  nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ ffmpeg_4 libxslt ];
+  nativeBuildInputs = [ pkg-config buildPackages.libxslt.bin ];
+  buildInputs = [ ffmpeg_4 ];
 
   meta = with lib; {
     homepage = "https://www.flameeyes.eu/projects/unpaper";

